src/pages/staticPages/tools/mutationMapper/MutationMapperTool.tsx

```diff
@@ -38,13 +50,23 @@ export default class MutationMapperTool extends React.Component<
     @observable inputFileContent: string | undefined;
     @observable showIncorrectInput = false;
     @observable lastParsedInputContent: string | undefined;
+    @observable referenceGenomeSelection: string = referenceGenome.GRCH37;
 
     private store: MutationMapperToolStore = new MutationMapperToolStore();
 
     constructor(props: IMutationMapperToolProps) {
         super(props);
 
         this.userSelectionStore = new MutationMapperUserSelectionStore();
+        // store genome build selection to localStorage, so it will not get lost after reload the page
+        if (getBrowserWindow().localStorage.getItem('genomenexus_url_grch38')) {
```

Comment:
is this just for testing purposes? the comment says "store" it in localStorage, but all i see is reading from localstorage.

Reply:
ok, i see.  this i a user setting